src/scrape_up/github/respository.py

```diff
@@ -403,4 +403,17 @@ def get_environment(self):
             return {
                 "data": None,
                 "message": message,
-            }
\ No newline at end of file
+            }
+
+    def get_branch(self):
+        data = requests.get(f"https://github.com/{self.username}/{self.repository}/branches")
+        data = BeautifulSoup(data.text, "html.parser")
+        try:
+            branch = data.find_all(class_="branch-name css-truncate-target v-align-baseline width-fit mr-2 Details-content--shown")
+            allBranches = []
+            for branchNames in branch:
+                allBranches.append(branchNames.text.strip())
+            return allBranches
+        except:
+            message = "Failed to fetch branches"
+            return message
```

Comment:
Please follow the return type structure of the project, check other methods for reference.

Reply:
Please review. I've updated the return type structure. @nikhil25803